external/watch/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
Forgot to make it a submodule. This implementation is extracted from Tokio and it's the same, but with no_std support. I made it a separate repo https://github.com/fobnail/watch and added as git dependency in coap-server-rs instead of submodule